src/ssl.c

```diff
@@ -15793,15 +15793,31 @@ void wolfSSL_DES_set_odd_parity(WOLFSSL_DES_cblock* myDes)
 }
 
 
+#ifdef WOLFSSL_DES_ECB
 void wolfSSL_DES_ecb_encrypt(WOLFSSL_DES_cblock* desa,
```

Comment:
I could not find wolfSSL_write_dup in src/ssl.c.
So I saw wolfSSL_DES_set_key_checked and write like this.

/* Encrpyt or decrypt input message desa with key and get output in desb. 
 * if enc is DES_ENCRYPT,input message is encrypted or
 * if enc is DES_DECRYPT,input message is decrypted.
 * */

if English is strange,please correct me.